.eslintignore

```diff
@@ -0,0 +1 @@
+analytics.js
```

Comment:
Can you add this into package.json ignorePatterns instead of adding a new file?
Or take that out and do it all this way, but let's not use two ways to ignore files for linting.